admin/server/users.go

```diff
@@ -284,6 +284,59 @@ func (s *Server) GetUser(ctx context.Context, req *adminv1.GetUserRequest) (*adm
 	return &adminv1.GetUserResponse{User: userToPB(user)}, nil
 }
 
+func (s *Server) DeleteUser(ctx context.Context, req *adminv1.DeleteUserRequest) (*adminv1.DeleteUserResponse, error) {
```

Comment:
The RPC here still relies on `req.Organization`, which is no longer passed from the CLI, so it can't work.